release-content/release-notes/camera_controllers.md

```diff
@@ -0,0 +1,51 @@
+---
+title: First-party camera controllers
+authors: ["@alice-i-cecile"]
+pull_requests: [20215]
+---
+
+To understand a scene, you must look at it through the lens of a camera: explore it, and interact with it.
+Because this is such a fundamental operation, game devs have developed a rich collection of tools
+called "camera controllers" for manipulating them.
+
+Getting camera controllers feeling *right* is both tricky and essential: they have a serious
+impact on both the feeling of your game and the usability of your software.
+
+Historically, Bevy has left this entirely up to individual game developers:
+camera controllers require deep customization and endless twiddling.
+However, Bevy as a game engine needs its *own* camera controllers:
+allowing users to quickly and easily explore scenes during development (rather than gameplay).
+
+To that end, we've created `bevy_camera_controller`: giving us a place to store, share and refine the camera controllers
+that we need for easy development, and yes, an eventual Editor.
+We're kicking it off with a couple of camera controllers, detailed below.
+
+### `FreeCam`
+
+The first camera controller that we've introduced is a "free cam", designed for quickly moving around a scene,
+completely ignoring both physics and geometry.
+You may have heard of a "flycam" controller before, which is a specialization of a "free cam" controller
```

Comment:
I'll be honest, I don't see the difference between them and this sentence makes me wonder why ours is a free cam and not a fly cam.